enterprise_learner_portal/utils.py

```diff
@@ -21,45 +17,32 @@ class CourseRunProgressStatuses:
 
 def get_course_run_status(course_overview, certificate_info, enterprise_enrollment):
     """
-    Get the progress status of a course run, given the state of a user's certificate in the course.
+    Get the status of a course run, given the state of a user's certificate in the course.
 
-    In the case of self-paced course runs, the run is considered completed when either the course run has ended
-    OR the user has earned a passing certificate 30 days ago or longer.
```

Comment:
What was the original motivation for this 30-day wait period?

Reply:
Good question. We tried to replicate some logic around course run status from edx-platform 17 months ago, which had that 30-day wait period built in.

This was for the `program_enrollments` endpoint. I checked with Joe about the 30-day wait period to see if there was a valid reason for it; he reached out to Marco who wasn't even aware of it haha.

So, no need for it here; it's not serving any purpose really other than causing UX issues :)